architecture/core_concepts/routes.adoc

```diff
@@ -817,11 +819,32 @@ resolution order (oldest route wins).
 ====
 
 
-[[route-specific-timeouts]]
-== Route-specific Timeouts
+[[route-specific-annotations]]
+== Route-specific Annotations
 
-By default, a server-side timeout is set across all routes. It is possible to
-change that timeout using an annotation in the route configuration.
+Using environment variables as defined in xref:env-variables[Configuration Parameters],
+a router can set the default options for all the routes it exposes. An individual route can
+override some of these defaults by providing specific configuration in its annotations.
+
+
+*Route Annotations*
+
+For all the items outlined in this section, you can set annotations on the
+*route definition* for the route to alter its configuration
+
+.Route Annotations
+[cols="2*", options="header"]
+|===
+|Variable | Description
+|`*haproxy.router.openshift.io/balance*`| sets the load balance algorithm. Available options are `source`, `roundrobin`, or `leastconn`.
+|`*haproxy.router.openshift.io/timeout*` | sets a server side timeout. overwrites `*ROUTER_DEFAULT_SERVER_TIMEOUT*` for that route
+|`*haproxy.router.openshift.io/rate-limit-connections*`| setting to true enables rate limiting functionality
+|`*haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp*`| limits the number of concurrent tcp connections shared by an IP address
+|`*haproxy.router.openshift.io/rate-limit-connections.rate-tcp*`| limits the rate at which an IP address can make tcp connections
+|`*haproxy.router.openshift.io/rate-limit-connections.rate-http*`| limits the rate at which an IP address can make http requests
+|`*haproxy.router.openshift.io/disable_cookies*`| disables the use of cookies, the balance algorithm is used to determine which backend serves connections everytime
```

Comment:
Disables the use of cookies to track related connections.  If set to 'true' the balance algorithm is used to choose which back-end serves connections for each HTTP request.